src/xtgeo/xyz/polygons.py

```diff
@@ -179,10 +433,14 @@ def dtname(self, value):
         super()._check_name(value)
         self._dtname = value
 
```

Comment:
```
>>> p = xtgeo.Polygons([[1,2,3],[1,2,3],[1,2,3]])
>>> p
Polygons(values=   X_UTME  Y_UTMN  Z_TVDSS  POLY_ID
0     1.0     2.0      3.0        0
1     1.0     2.0      3.0        0
2     1.0     2.0      3.0        0, xname=X_UTME, yname=Y_UTMN, zname=Z_TVDSS, pname=POLY_ID, hname=H_CUMLEN, dhname=H_DELTALEN, tname=T_CUMLEN, dtname=T_DELTALEN, name=poly, 
>>> p.dataframe
   X_UTME  Y_UTMN  Z_TVDSS  POLY_ID
0     1.0     2.0      3.0        0
1     1.0     2.0      3.0        0
2     1.0     2.0      3.0        0
>>> p.hname
'H_CUMLEN'
>>> p.hname = "CC"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jan/work/git/xtg/xtgeo/src/xtgeo/xyz/polygons.py", line 392, in hname
    super()._check_name(value)
  File "/home/jan/work/git/xtg/xtgeo/src/xtgeo/xyz/_xyz.py", line 107, in _check_name
    raise ValueError(
ValueError: CC does not exist as a column name, must be one of: fIndex(['X_UTME', 'Y_UTMN', 'Z_TVDSS', 'POLY_ID'], dtype='object')
>>> 
```

Reply:
Isn't that the current behavior?

Reply:
Perhaps best to wait with that change until 3.0 so that we preserve semver.

Reply:
Or if it is buggy behavior then perhaps we should change it in a separate PR.

Reply:
OK, we can take it later